Source/Mosa.Demo.SVGAWorld.x86/Boot.cs

```diff
@@ -121,8 +134,6 @@ private static void DoGraphics()
 				() => { PCService.Reset(); return null; }));
 			Taskbar.Buttons.Add(new TaskbarButton(Taskbar, "Paint", Color.Coral, Color.White, Color.Red,
 				() => { WindowManager.Open(new Paint(70, 90, 400, 200, Color.MediumPurple, Color.Purple, Color.White)); return null; }));
-			Taskbar.Buttons.Add(new TaskbarButton(Taskbar, "Notepad", Color.Coral, Color.White, Color.Red,
```

Comment:
I modified the GraphicalConsole class (now GraphicalTextDevice), which, because it uses ITextDevice, now takes up the whole screen and can no longer be used as a "regular control". Because I was honestly a bit lazy, I didn't really want to re-implement the notepad (but it would be pretty straight forward to implement).